packages/cli/src/cmd/add.ts

```diff
@@ -41,57 +65,99 @@ async function addApp(props: {
   // If no type specified, show interactive select
   let appType = type;
   if (!appType) {
+    const allTypes = [...APP_TYPES, ...PACKAGE_TYPES];
     const { selected } = await inquirer.prompt([
       {
         type: "list",
         name: "selected",
-        message: "Which app type would you like to add?",
-        choices: APP_TYPES.map((t) => ({ name: t.name, value: t.value })),
+        message: "Which type would you like to add?",
+        choices: allTypes.map((t) => ({ name: t.name, value: t.value })),
       },
     ]);
     appType = selected;
   }
 
-  if (appType !== "next") {
+  if (!["next", "vite", "pkg"].includes(appType)) {
     printComingSoon(appType);
     return;
   }
 
-  // Ask for app name if not provided
-  let appName = nameArg;
-  if (!appName) {
+  // Determine template type
+  let templateType = "app";
+  if (appType === "pkg") {
+    templateType = "package";
+  }
+
+  // Ask for name if not provided
+  let itemName = nameArg;
+  if (!itemName) {
+    const itemTypeLabel = templateType === "package" ? "package" : "app";
     const { inputName } = await inquirer.prompt([
       {
         type: "input",
         name: "inputName",
-        message: "What is the name of your app?",
-        validate: (input: string) => input ? true : "App name is required",
+        message: `What is the name of your ${itemTypeLabel}?`,
+        validate: (input: string) => input ? true : `${itemTypeLabel} name is required`,
       },
     ]);
-    appName = inputName;
+    itemName = inputName;
   }
-  const appSlug = slugify(appName);
+  const appSlug = slugify(itemName);
 
-  // Determine where to place the new app directory
+  // Determine where to place the new directory
   let destDir: string;
   const cwd = process.cwd();
   const cwdBase = path.basename(cwd);
-  if (cwdBase === "apps") {
-    destDir = path.resolve(cwd, appSlug);
+  
+  if (templateType === "package") {
```

Comment:
No, we should be using degit to clone a template from the git repo. This should never be local